hack/deployer/config/plans.yml

```diff
@@ -64,6 +64,7 @@ plans:
   ocp:
     region: europe-west2
     nodeCount: 3
+    overwriteDefaultKubeconfig: true
```

Comment:
Should we be explicit in `ocp-dev` and set `overwriteDefaultKubeconfig: false`?

Reply:
not a bad idea, I'll do it if we need another push to this PR. Otherwise, I'll send a follow-up patch :)

Reply:
done!